chromium_src/ui/webui/resources/cr_components/color_change_listener/colors_css_updater.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { ColorChangeUpdater, COLORS_CSS_SELECTOR } from './colors_css_updater-chromium.js';
+
+const DEFAULT_COLORS_HREF = '//theme/colors.css?sets=ui';
+
+const oldStart = ColorChangeUpdater.prototype.start;
+
+ColorChangeUpdater.prototype.start = function() {
+  const root: ShadowRoot | Document = (this ).root_
+  // Ensure that we load colors.css or the ColorChangeUpdater won't update them
+  // when the theme changes.
```

Comment:
I think this could do with a little more of a comment as to why this is required for Brave and not for Chromium

Reply:
Done!